Backend/NoCode.FlowerShop.Api/Controllers/FlowersController.cs

```diff
@@ -15,4 +19,16 @@ public FlowersController(ISender mediator, IMapper mapper)
         _mediator = mediator;
         _mapper = mapper;
     }
+
+    [HttpPost("")]
+    [AuthorizeRoles(UserRole.Administrator)]
+    public async Task<IActionResult> Create(CreateFlowerRequest request)
+    {
+        var query = _mapper.Map<CreateFlowerCommand>(request);
```

Comment:
Command, not query.
```csharp
var command= _mapper.Map<CreateFlowerCommand>(request);
var result = await _mediator.Send(command);
 ```

Reply:
Will address in other PR on this controller